### PR TITLE
[#99] [Final] Write UITest for the Home screen

### LINKED ISSUE #99: Write UI Test for available screens in the app
## Why

We need to write  UI Test for available screens in the app to make sure it displays as expected

## Who Benefits?

Developer

### PR DESCRIPTION
**Pull request by markgravity**
https://github.com/markgravity/kmm-ic/issues/99

## What happened 👀

Write UITest for the Home screen

## Insight 📝

n/a

## Proof Of Work 📹

CleanShot 2023-05-18 at 10 51 18

### CODE REVIEW

File: shared/src/commonMain/kotlin/co/nimblehq/mark/kmmic/data/service/auth/AuthService.kt [hunk 1]
```diff
@@ -29,4 +32,14 @@ internal class AuthServiceImpl: AuthService, KoinComponent {
             }
         )
     }
+
+    override fun refreshToken(body: AuthRefreshTokenParams): Flow<AuthTokenApiModel> {
```
Comment (suho) [inline shared/src/commonMain/kotlin/co/nimblehq/mark/kmmic/data/service/auth/AuthService.kt]: Why we have this function in UITest PR

File: shared/src/commonMain/kotlin/co/nimblehq/mark/kmmic/data/service/api/ApiService.kt
Comment (phongvhd93) [inline shared/src/commonMain/kotlin/co/nimblehq/mark/kmmic/data/service/api/ApiService.kt:1]: Please check your rebase

File: shared/src/commonMain/kotlin/co/nimblehq/mark/kmmic/data/service/auth/AuthService.kt (hunk 1 above)
Comment (markgravity) [reply shared/src/commonMain/kotlin/co/nimblehq/mark/kmmic/data/service/auth/AuthService.kt]: @suho rebased the branch 🙏

File: shared/src/commonMain/kotlin/co/nimblehq/mark/kmmic/data/service/api/ApiService.kt
Comment (markgravity) [reply shared/src/commonMain/kotlin/co/nimblehq/mark/kmmic/data/service/api/ApiService.kt:1]: @phongvhd93  rebased the branch 🙏

**Review by phongvhd93 — APPROVED** [review]

**Review by suho — APPROVED** [review]

**Pull request merged**